release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -117,6 +117,12 @@ Ingress Node Firewall Operator was made a technology preview in {product-title}
 
 // Note: use [discrete] for these sub-headings.
 
+[discrete]
+[id="ocp-4-14-cert-manager-operator-1-11-1"]
+=== cert-manager Operator general availability
+
+{cert-manager-operator} 1.11.1 is generally available in {product-title} 4.14.
```

Comment:
This is right, but this is confusing easily making readers think the meaning appears it is GA starting with 4.14.

Reply:
@xingxingxia Updated the section. This [1] information would suffice, right? Previous release notes are not mentioning the cert-manager Operator version, so I did not mention that here. 
[1]
```{cert-manager-operator} was introduced as a Technology Preview in {product-title} 4.11 and is now generally available in {product-title} 4.14.```